src/generator/css/base.less

```diff
@@ -803,3 +806,68 @@ body.post-page {
         overflow-wrap: break-word;
     }
 }
+
+
+/** People page */
+
+.people-list-wrap {
+    @apply text-center lg:max-w-7xl;
+
+    .people-list {
+        @apply grid sm:grid-cols-3 py-0;
+        width: 100%;
+        list-style-type: none;
+        padding: initial;
+
+        .person-body {
+            @apply my-5 text-center;
+            padding: 0 var(--layout-spacing-v-half);
+
+            a {
+                text-decoration: none;
+            }
+
+            a:hover {
+                cursor: pointer;
+            }
+
+            .image {
+                @apply mb-5 lg:block md:mb-0;
+
+            }
+        }
+    }
+}
+
+
+/**Individual Person page*/
+.person-page {
+    .person {
+        @apply grid mt-16 mb-16 align-text-top text-center place-items-center;
+        padding: 0 var(--layout-spacing-horizontal);
```

Comment:
Just for future reference, no need to refactor
Tip for working with grids: use "gap" in css: https://developer.mozilla.org/en-US/docs/Web/CSS/gap